pkg/tests/end_to_end_tests/main_test.go

```diff
@@ -75,6 +76,8 @@ func setExtensionState() {
 	case 12:
 		extensionState.UsePG12()
 	case 13:
+		extensionState.UsePG13()
+	case 14:
```

Comment:
Nit: maybe worth commenting that `14` is the default extension state so no bit manipulation needed.